docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -72,6 +72,15 @@ For more complex or time-consuming searches, you can use <<async-search>> with
 {search-snaps}.
 ====
 
+[[searchable-snapshots-repository-types]]
+You can use any of the following repository types with searchable snapshots:
+
+* {plugins}/repository-s3.html[AWS S3]
```

Comment:
Should we state "and other fully S3-compatible blob stores (Minio, ...)" as well as generally link to the repo analyzer?

Reply:
That was in an earlier draft but I struggled to find a good way to express it. You're right tho, mentioning this will cover a whole other class of questions. See ddd25d16bd1.

Reply:
I'll open a separate PR to link to the analyze API since that will only target 7.12+.